editor/filesystem_dock.cpp

```diff
@@ -2588,6 +2588,14 @@ void FileSystemDock::_file_option(int p_option, const Vector<String> &p_selected
 			String dir = ProjectSettings::get_singleton()->globalize_path(fpath);
 			ScriptEditor::get_singleton()->open_text_file_create_dialog(dir);
 		} break;
+		case FILE_MENU_RUN_SCRIPT: {
+			if (!p_selected.is_empty()) {
```

Comment:
```suggestion
			if (p_selected.size() == 1) {
```
Does not make sense for multiple files IMO.